config/formats/typescriptExportDefinition.test.ts

```diff
@@ -92,17 +112,53 @@ describe('Format: TypeScript definitions', () => {
         tokens: {
           subgroup: {
             red: ColorHex;
-            yellow: string;
+            stringValue: string;
             border: Border;
-            complex: {
-              top: number;
-              bottom: any;
-            };
+            numberValue: number;
+            booleanValue: "boolean";
           };
         };
       };`,
       {parser: 'typescript', printWidth: 500}
     )
     expect(typescriptExportDefinition(input)).toStrictEqual(expectedOutput)
   })
+
+  // it('throws an invalidToken error if a token has no value property', () => {
```

Comment:
should this be disabled?

Reply:
Yes, I removed the part as it does not throw an error anymore. Items without a `value` property get ignored now. I wrote this down somewhere in slack, but i should have rather added it here. :D